.github/workflows/rector.yaml

```diff
@@ -21,6 +24,26 @@ jobs:
       - name: "Checkout code"
         uses: "actions/checkout@v2"
 
+      - name: "Set target PHP version to 7.3"
+        if: "github.event.inputs.phpVersion == '7.3'"
+        run: "sed -i -e 's#//PHP73 ##' rector.php"
+
+      - name: "Set target PHP version to 7.2"
+        if: "github.event.inputs.phpVersion == '7.2'"
+        run: "sed -i -e 's#//PHP72 ##' rector.php"
+
+      - name: "Set target PHP version to 7.1"
+        if: "github.event.inputs.phpVersion == '7.1'"
+        run: "sed -i -e 's#//PHP71 ##' rector.php"
+
+      - name: "Set target PHP version to 7.0"
+        if: "github.event.inputs.phpVersion == '7.0'"
+        run: "sed -i -e 's#//PHP70 ##' rector.php"
+
+      - name: "Set target PHP version to 5.6"
+        if: "github.event.inputs.phpVersion == '5.6'"
+        run: "sed -i -e 's#//PHP56 ##' rector.php"
```

Comment:
downgrading to 7.0 and 5.6 is not really feasible right now, the number of rules to downgrade to them is very tiny. PHP 7.1 is the highest you can reliably downgrade to

Reply:
The thing is I do not really know what Rector is.